src/renderer/components/Modal/ModalCloseConfirmation.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <div
```

Comment:
Out of curiosity, how come you didn't use the ModalWindow component as the other modals do? I think it may be worth making them consistent

Reply:
I think I know. Is it because the confirm modal is shown inside ModalWindow?

Reply:
Yep. We cannot use `ModalWindow` inside `ModalWindow`. I was expecting to use `ModalConfirmation`, but it also uses `ModalWindow`. Modals cannot stack (see #1651), so I have to create an entire component isolated from the rest.